operators/cuda/cuda_ops.cc

```diff
@@ -31,13 +32,15 @@ FxLoadCustomOpFactory LoadCustomOpClasses_Contrib = []() -> CustomOpArray& {
       CustomCudaStructV2("FastGelu", contrib::FastGelu<float>),
       CustomCudaStructV2("MulSharedInput", MulSharedInputFloat32Type),
       CustomCudaStructV2("NegXPlus1", contrib::NegXPlus1<float>),
+      CustomCudaStructV2("ReplaceZero", contrib::ReplaceZero<float>),
```

Comment:
Not sure how the kernel selection works. Do we need `half`, `int64_t`, etc. as template parameters?

Reply:
All templates are implemented for float and float16 because that's the only types i need. Adding bfloat16 makes sense as well. Integer does not really make sense. I kept the list short to avoid growing the binary. I can add bfloat16 if you think that makes sense.